Examples/cChocoFeatureExample.ps1

```diff
@@ -0,0 +1,28 @@
+
+
+configuration ChocoFeatures {
+
+    Import-DscResource -ModuleName cChoco
+
+    Node 'localhost' {
+
+        cChocoFeature allowGlobalConfirmation {
+
+            FeatureName = "allowGlobalConfirmation"
```

Comment:
Hmm interesting idea, to ensure I'm clear - do you mean having the name of the resource be used as the FeatureName?  IE simply having the below to configure allowGlobalConfirmation as present. 

```
cChocoFeature allowGlobalConfirmation {
}
```

I'll do some research on this, I've not seen this done before - and as FeatureName is acting as the key i think it will need to be explicitly stated.